PySDM/particulator.py

```diff
@@ -72,6 +72,9 @@ def Random(self):
     def n_sd(self) -> int:
         return self.__n_sd
 
+    def n_sd_setter(self, value):
```

Comment:
The name suggests setting (i.e., replacing old value), but the function is adding.
There should be no need to alter the `__n_sd` since the number of SD does not change throughout simulation.
I don't see this function being used anywhere on this branch also, right?